<launch>
  <param name="function" value="1" />
  <param name="ec_cluster_tolerance" value="0.0035" />
  <param name="ec_min_cluster_size" value="1000" />
  <param name="ec_max_cluster_size" value="10000" />
  <param name="crop_min_x" value="0" />
  <param name="crop_max_x" value="1.3" />
  <param name="crop_min_y" value="-0.34" />
  <param name="crop_max_y" value="-0.07" />
  <param name="crop_min_z" value="0.99" />
  <param name="crop_max_z" value="1.2" />
  <node pkg="perception" type="point_cloud_demo" name="point_cloud_demo" output="screen">
    <remap from="cloud_in" to="mock_point_cloud" />
  </node>
</launch>
